src/tasks/mt.py

```diff
@@ -65,12 +65,13 @@ def load_data(self, path):
                 row = row.strip().split('\t')
                 if len(row) < 2 or not row[0] or not row[1]:
                     continue
-                src_sent = process_sentence(row[0], self.max_seq_len)
+                src_sent = process_sentence(row[0], self.max_seq_len, tokenizer=self._tokenizer_name)
                 # target sentence sos_tok, eos_tok need to match Seq2SeqDecoder class
+                # NOTE(Alex): don't use BERT tokenization on targets
```

Comment:
Is it worth passing `tokenizer="MosesTokenizer"` or similar to make this more explicit?